packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js

```diff
@@ -0,0 +1,55 @@
+const _ =  require('underscore');
+const loginEnBundle = require('../../../i18n/src/json/login.json');
+const { existsSync, readFileSync } = require('fs-extra');
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect missing i18nkeys from json mock files',
+      category: 'i18n Issues',
+      recommended: true,
+    },
+    messages: {
+      missingApiKeysLoginBundle: 'Missing i18n key from login.properties returned from the API. Add {{i18nKey}} to login.properties to prevent english leaks',
+      missingi18nKeyApiResponse: 'Missing i18n key in API mock {{file}}. Avoid using hard-coded strings returned and instead have the API return an i18n key and add that key to login.properties to prevent english leaks'
```

Comment:
Let's try to keep these error messages as short as possible. Consider rewording to:

```suggestion
      missingApiKeysLoginBundle: '"{{i18nKey}}" is missing from login.properties.',
      missingi18nKeyApiResponse: 'API mock "{{file}}" does not have an i18nKey. Avoid hard-coding English strings.'
```
Ideally, we'll have docs for these rules to outline how developers can **fix** these issues. Can we add a `docs` section to this package and/or create a follow up ticket to do so?

Reply:
I will update error and add a follow up JIRA

Reply:
https://oktainc.atlassian.net/browse/OKTA-389342